project_app/components/ToolBar.js

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+function ToolBar() {
+  return (
+    <Container>
+      <div className="search">
+        <div>
+          <input type="text" />
+          <h1>Search board game </h1>
```

Comment:
as i said in discord
```suggestion
          <input type="text" placeholder="Search board game" />
```
and make sure it's typeable, and the text is not white